package.json

```diff
@@ -12,6 +12,7 @@
   ],
   "scripts": {
     "build": "tsc -p .",
+    "build-watch": "nodemon --exec \"yarn build\" --watch src --ext ts",
```

Comment:
I'd rather use the built-in watch mode, instead of running `yarn build` on every change.

```
tsc -p . -w
```

And `yarn dev` is more common for scripts like this, rather than `yarn build-watch`.